common/scripts/mesh-ibss.sh

```diff
@@ -90,8 +90,18 @@ echo "sudo mesh $1 $2 $3 $4 $5 $6 $7 $8 $9 ${10} ${11}"
         help
       fi
 
-cat <<EOF >/var/run/wpa_supplicant-adhoc.conf
-ctrl_interface=DIR=/var/run/wpa_supplicant
+
+phy_if=$(iw dev $wifidev info | awk '/wiphy/ {printf "phy" $2}')
+rsn_ibss_cap=$(iw phy $phy_if info | awk '/RSN/ {printf $3}')
+
+if [ "$rsn_ibss_cap" = "RSN-IBSS." ]; then
```

Comment:
Even qca6174 reports RSN-IBSS support, not working.  Do not merge before this is solved!

Reply:
Yeah, we need to fix ath10k bug.
If its FW bug, it will be long pole to fix.
Right now iwl and ralink radio is tested.

Reply:
I am able to repro with QCA6174<------>ralink and RM4.4.XXX FW is drooping the EAPOL frame(M1).
Same observation is seen with the logs shared by you.
I have initiated one thread in ath10k upstream group, we can revisit this once we have fix available for QCA6174 or we can explicitly enable to known working radios(in future).

http://lists.infradead.org/pipermail/ath10k/2021-August/012738.html